docs/docs/guides/how-to-seed-mysql.md

```diff
@@ -0,0 +1,209 @@
+---
+title: How to programmatically seed MySQL databases for integration tests
+sidebar_label: Programmatically seeding MySQL databases
+slug: /how-to-seed-mysql
+toc_max_heading_level: 2
+sidebar_position: 6
+---
+
+Introduction
+------------
+In this guide, you will programmatically seed a MySQL database for integration tests in an ExpressJS application. ExpressJS can be replaced with any containerized microservice framework you wish to use.
```

Comment:
Reading through the entire How To guide, i don't believe we use ExpressJS at all. Can we take this out?